webclient/app/src/app/features/config/MitigationActionTypeOverview.jsx

```diff
@@ -1,4 +1,176 @@
+import {Button, MenuItem, Select, Stack, Typography} from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import SaveIcon from '@mui/icons-material/Save';
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import {DataGrid, GridActionsCellItem} from '@mui/x-data-grid';
+import {useTranslation} from "react-i18next";
+import React, {useEffect, useState, useMemo} from "react";
+import MitigationActionTypeRest from "../../services/MitigationActionTypeRest";
+
+const DropdownMenu = ({row, updateRow}) => {
```

Comment:
das würde ich in die Komponente packen. Was ist das? Eine eigene Komonente? Die Notation finde ich unleserlich